docs/source/model_design/vivarium_features/automated_v_and_v/index.rst

```diff
@@ -0,0 +1,278 @@
+  CI server impossible.
+
+Tests have been developed by the engineering team to cover most of the simulation's functionality,
+though test coverage is not complete.
+Notably, none of the observers currently have integration tests.
+
+Fuzzy checking
+--------------
+
+It is fairly straightforward to automate checks such as
+"a household *always* has exactly one living reference person."
+It is more difficult to check that a value is correct when the value in the
+simulation is subject to stochastic variation.
+We clearly cannot check that the rate of events in the simulation is **exactly** the expected rate,
+or that a mean value of a continuous risk factor is **exactly** as expected.
+Nor can we use an approximation like "within X% of the target value,"
+since how much a value deviates due to chance depends on our population size.
+The difficulty of this problem is part of why, in the manual V&V process, we usually check such values visually.
+
+Note that fuzzy checking can be applied to both **verification** and **validation**.
+For verification, the "target" is that the simulation's value is exactly
+correct.
+For validation, we specify a target range, within which we expect the simulation's **true** value (i.e. the value of the simulation result as the simulated population size goes to infinity) should fall.
+For example, we could specify that the simulation's true value should be within 10% of the GBD estimate, which means it is within 10% **not including stochastic variation.**
+
+We have begun to formalize fuzzy checking using frequentist hypothesis tests,
+one for each of the values we want to check in the simulation.
+In these hypothesis tests, the null hypothesis is that the simulation value matches the V&V target;
+rejecting the null hypothesis indicates a problem with the simulation.
+
+Sensitivity and specificity
++++++++++++++++++++++++++++
+
+The **sensitivity** of a check is the probability of it catching
+an issue, given that the issue is present.
+The **specificity** is the probability of the check passing when
+there is no issue present.
+
+In this diagram `from Wikipedia <https://en.wikipedia.org/wiki/Sensitivity_and_specificity>`_, the yellow plane represents the decision boundary: to the left of this boundary, our check
+considers the simulation "Healthy"; to the right, our check
+considers the simulation "Sick."
+The boundary can be chosen arbitrarily, which illustrates the direct tradeoff
+between sensitivity and specificity.
+By moving it left, we reduce the number of false negatives (missed bugs, in the simulation context), increasing sensitivity.
+However, we also increase the number of false positives (false alarms),
+decreasing specificity.
+
+.. figure:: PPV,_NPV,_Sensitivity_and_Specificity.svg
+
+  By Original by Luigi Albert Maria - SVG version of File\:PPV, NPV, Sensitivity and Specificity.pdf, CC BY-SA 4.0, https://commons.wikimedia.org/w/index.php?curid=99283192
```

Comment:
```suggestion
  Original by Luigi Albert Maria - SVG version of File\:PPV, NPV, Sensitivity and Specificity.pdf, CC BY-SA 4.0, https://commons.wikimedia.org/w/index.php?curid=99283192
```

Reply:
Yep I did notice this, but since the typo [is on their end](https://commons.wikimedia.org/w/index.php?curid=99283192) (it literally says "Original by Luigi Albert Maria" in the "Author" field) I figured I would keep as-is.